Jenkins/update_ci_win_docker_image.ps1

```diff
@@ -0,0 +1,83 @@
+
+$ErrorActionPreference = "Stop"
+
+$globalVariables = (Resolve-Path "$PSScriptRoot\..\global-variables.ps1").Path
+$ciUtils = (Resolve-Path "$PSScriptRoot\..\Modules\CIUtils").Path
+
+Import-Module $ciUtils
+. $globalVariables
+
+$WORKSPACE="$env:WORKSPACE"
```

Comment:
Please use the environment variable directly. It's not needed to create a new variable.